docs/articles/documentation/reference/configuration-file.md

```diff
@@ -1032,6 +1033,21 @@ You can also disable page caching for an individual [fixture](test-api/fixture/d
 *CLI*: [--disable-page-caching](command-line-interface.md#--disable-page-caching)  
 *API*: [runner.run({ disablePageCaching })](testcafe-api/runner/run.md)
 
+## disableMultipleWindows
+
+Disables support for [multi-window testing](../guides/advanced-guides/multiple-browser-windows.md).
+
+```json
+{
+    "disableMultipleWindows": true
+}
+```
+
+The `disableMultipleWindows` option disables support for multi-window testing in Chrome and Firefox. Use this setting if you encounter compatibility issues with [TestCafe v2015.1](http://testcafe.devexpress.com/documentation) tests.
```

Comment:
> Use this setting if you encounter compatibility issues with [TestCafe v2015.1](http://testcafe.devexpress.com/documentation) tests.

Here and below.
It's related to all test formats (not only for TestCafe v2015.1 tests).